Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -244,6 +244,32 @@ def metadata() -> pd.DataFrame:
     return meta_df
 
 
+def _detect_metadata(data: pd.DataFrame,
+                     data_source_col: str = "data_source",
+                     signal_col: str = "signal",
+                     geo_type_col: str = "geo_type") -> Tuple:
+    """Given a DataFrame, return the signal attributes of that DataFrame.
+
+    Errors if any of the columns are heterogeneous.
+
+    :param data: dataframe with geo_value an
```

Comment:
did you accidentally a word?

Reply:
Oops, fixed. I don't know if your comment was intentional or not lol.

Reply:
https://knowyourmeme.com/memes/i-accidentally